jstreegrid.js

```diff
@@ -456,11 +457,25 @@
 					findDataCell(this.uniq,removeNodes,this._gridSettings.gridcols).remove();
 				}
 			}, this))
-			.on("hide_node.jstree", $.proxy(function (e, data) {
-				// remove out own data cells
-        findDataCell(this.uniq, data.node.id, this._gridSettings.gridcols).remove();
-				// hide any children
-        this._hide_grid(data.node);
+      .on("hide_node.jstree", $.proxy(function (e, data) {
+        // remove out own data cells
+        findDataCell(this.uniq, data.node.id, this._gridSettings.gridcols).removeClass("jstree-grid-hidden");
+        if (data.node.state.opened) {
```

Comment:
1. Do we need to recurse down? What if it goes down multiple layers?
2. Do we need to look at children at all? Shouldn't we get events for the children?